projects/sandbox/datagen/datagen/utils/timeslide_waveforms.py

```diff
@@ -72,14 +83,20 @@ def calc_segment_injection_times(
     Calculate the times at which to inject signals into a segment
 
     Args:
-        start: The start time of the segment
-        stop: The stop time of the segment
-        spacing: The spacing between signals
-        jitter: The jitter to apply to the signal times
-        buffer: The buffer to apply to the start and end of the segment
-        waveform_duration: The duration of the waveform
-
-    Returns np.ndarray of injection times
+        start:
+            The start time of the segment
+        stop:
+            The stop time of the segment
+        spacing:
+            The spacing between signals in seconds
+        jitter:
```

Comment:
Maybe more description of what `jitter` and `buffer` are doing and why they're there